recipes/libsemigroups-python-bindings/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "libsemigroups-python-bindings" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "7764bedf64503dcb10679394d8b0df8581b791829e1aebee70bc8cf31f4309f2" %}
+{% set repo = "https://github.com/james-d-mitchell/libsemigroups" %}
+{% set home = "https://james-d-mitchell.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The python bindings are not yet merged in master; fetch them from their git
+  # branch instead
+  #git_url: {{ repo }}.git
+  #git_tag: python-bindings
+  url: {{ repo }}/archive/conda-test.tar.gz 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+    - cd python-bindings
+    - python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - cython
+    - libsemigroups >=0.2.2
+    - pytest-runner
+    # pytest-cython # not yet on conda-forge
+
+  run:
+    - python
+    - libsemigroups >=0.2.2
+
+test:
+  requires:
+    - pytest
+  commannds:
```

Comment:
You can't run `py.test` since `pytest-cython` is not packages yet. So comment the `commands:` part for now